telemetry/telemetry.go

```diff
@@ -0,0 +1,37 @@
+package telemetry
+
+import (
+	"github.com/gruntwork-io/go-commons/telemetry"
+	"os"
+	"strings"
+)
+
+var sendTelemetry = true
+var telemetryClient telemetry.MixpanelTelemetryTracker
+var cmd = ""
+var isCircleCi = false
+var account = ""
+
+func InitTelemetry(name string, version string, clientId string) {
+	_, disableTelemetryFlag := os.LookupEnv("DISABLE_TELEMETRY")
+	isCircleCi = os.Getenv("CIRCLECI") == "true"
+	clientIdExists := clientId != ""
+	sendTelemetry = !disableTelemetryFlag && clientIdExists
+	if sendTelemetry {
+		cmd = strings.Join(os.Args[1:], " ")
+		telemetryClient = telemetry.NewMixPanelTelemetryClient(clientId, name, version)
+	}
+}
+
+func SetAccountId(accountId string) {
```

Comment:
This function appears unused.

Reply:
It's in aws.go on line 230.  I set it as soon as it is possible to call `sts.getcalleridentity`